src/main/ruby/truffleruby/core/truffle/concurrent_hash_map.rb

```diff
@@ -0,0 +1,146 @@
+# frozen_string_literal: true
+
+module TruffleRuby
+  class ConcurrentHashMap
+    def initialize(options = nil)
+      @hash = {}
+    end
+
```

Comment:
You need to implement `initialize_copy`, that's probably what's needed for passing the dup specs.